web/sites/all/modules/onlinekurslabor/custom_general/css/custom.css

```diff
@@ -1302,22 +1444,53 @@ div.view-courses-kurse-bersicht .courses_small_containerhome_course_item  img.co
 /* draft-label fuer kurse */
 .view_courses_label
 {
-    color:white; 
+    color:white;
     font-weight:bold;
     text-align:center;
     transform: rotate(-30deg);
     width: 90px;
-    position:absolute; 
+    position:absolute;
     left: 0px;
     top: 20px;
-    z-index:2 
+    z-index:2
 }
 .view_courses_label.draft
 {
       background: #D9534F;
 }
-.view_courses_label.unconfirmed
-{
-      background: orange;
+
+.view_courses_label.draft {
```

Comment:
das is jetzt doppelt

Reply:
https://github.com/n-multimedia/onlinekurslabor/commit/9e46464ba53b25848eb6d66c90366a0ecde608be